### PR TITLE
Add option to toggle exchanges sync

### LINKED ISSUE #2853: Way to enable/disable Sync with exchanges
## Abstract
At the moment rotki syncs the exchanges every time automatically and you have no direct control about sync.
A way to disable and enable the sync to specified exchanges would be great.

## Motivation

A problem that can occur is that you get IP banned because to many syncs.
So it would be helpful to disable the Balance or Trade sync to the exchanges.

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #2853

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.
- [ ] Add option to toggle exchanges sync
image
- [ ] Compress images on usage guide page

### CODE REVIEW

**Comment by yabirgb** [comment]
@lukicenturi do you think that putting the toggle as the first element in the row would improve the UI? Right now it feel a bit odd for me but could be just a personal taste

**Review by kelsos — APPROVED** [review]
The code looks good. Is there any chance we could make the description of the required action go to the second line and make it a bit smaller so that the column is closer to the actions?

**Review by LefterisJP — COMMENTED** [review]
Looks okay to me but still needs:

1. Changelog entry
2. Update usage guide to reflect that disabling sync with a specific exchange is possible,

**Review by LefterisJP — COMMENTED** [review]
Also wanna echo what @kelsos and @yabirgb said to make it somehow closer to the actions, to show that this is more of an actionable change rather than a status?

**Comment by lukicenturi** [comment]
@LefterisJP @yabirgb @kelsos 
I think the UI is better now.
I also updated the changelog and usage_guide
Please correct my grammar :(
image

File: docs/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature`2853` Add option to disable sync for connected exchanges.
```
Comment (yabirgb) [inline docs/changelog.rst:5]:
```suggestion
* :feature:`2853` Add option to disable sync for connected exchanges.
```

**Review by yabirgb — COMMENTED** [review]
Why in the changelog all the images have been edited?

**Comment by lukicenturi** [comment]
> Why in the list of changes appears as if all the images have been edited?

It's okay nothing changed. I just compress the size.

File: docs/changelog.rst (hunk 1 above)
Comment (lukicenturi) [reply docs/changelog.rst]: Thank you, updated @yabir

**Review by LefterisJP — APPROVED** [review]
LGTM thanks

**Pull request merged**